modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -99,18 +99,50 @@ The contents of the file resemble the following example:
 ----
 // An additional step for following this procedure when using oc-mirror as part of the disconnected install process.
 ifdef::oc-mirror[]
-. Save the file either as `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
-endif::[]
-
-ifdef::oc-mirror-v2[]
-. Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
+. Save the file as either `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
```

Comment:
ISG:
```suggestion
. Save the file as either `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`:
```

Reply:
This lead-in language strongly suggests that I'm about to follow mututally exclusive steps based on what I save my file as.